locales/de-DE.json

```
@@ -62,25 +89,40 @@
     "toggle_zen_mode": "Zen-Modus ändern"
   },
   "common": {
+    "confirm_dialog": {
+      "cancel": "Nein",
+      "confirm": "Ja",
```

Maybe this needs to be

```suggestion
      "confirm": "OK",
```